src/main/rx/flysky.c

```diff
@@ -206,8 +206,8 @@ static void checkRSSI (void)
 
     rssi_dBm = 50 + sum / (3 * FLYSKY_RSSI_SAMPLE_COUNT); // range about [95...52], -dBm
 
-    int16_t tmp = 2280 - 24 * rssi_dBm;// convert to [0...1023]
-    setRssiFiltered(constrain(tmp, 0, 1023), RSSI_SOURCE_RX_PROTOCOL);
+    int16_t tmp = 2280 - 24 * rssi_dBm; // convert to [0...1023]
+    setRssiDirect(tmp, RSSI_SOURCE_RX_PROTOCOL);
```

Comment:
Same question about removing the `constrain`

Reply:
The original source has constraints where RSSI is set, and where it is used. I have removed the constraints where the output scaling does not use the full range of the type used. I figure it's better to allow the RSSI display to go over 100% than to clip and not show the maximum value.